src/core/dom/layout/viewport-observer.js

```diff
@@ -49,14 +50,27 @@ export function observeWithSharedInOb(element, viewportCallback) {
   );
 }
 
+/**
+ * Unobserve an element.
+ * @param {!Element} element
+ */
+export function unobserveWithSharedInOb(element) {
```

Comment:
Who calls this?

Reply:
This is still called by various components that depend on the legacy `observeWithSharedInOb`/`unobserveWithSharedInOb` methods. I have #36088 to track this effort.